dataland-frontend/tests/e2e/specs/user-authentication/Register.ts

```diff
@@ -73,4 +74,45 @@ describe("As a user I want to be able to register for an account and be able to
     });
     logout();
   });
+
+  it("Checks that TOTP-Based 2FA can be enabled for the newly registered account", () => {
+    cy.task("getEmail").then((returnEmail) => {
+      cy.task("getPassword").then((returnPassword) => {
+        const username = returnEmail as string;
+        const password = returnPassword as string;
+        login(username, password);
```

Comment:
The login process here is a duplicate of the "Checks that one can login to the newly registered account" test above.
So in my opinion, we can drop the test above now to avoid unnecessary testing.
Do you agree?

Reply:
Yes and No. I think these are 2 different user flows which should be tested separetly

Reply:
Ok.